.codecov.yml

```diff
@@ -7,7 +7,7 @@ coverage:
   # https://docs.codecov.io/docs/commit-status
   status:
     # TODO: re-enable patch in the future
-    patch: off
+    patch: false
```

Comment:
Is `false` really accepted by Codecov?

I see their docs use `off` as well: https://docs.codecov.com/docs/commit-status#disabling-a-status

Reply:
It should, as `off` is part of the [YAML specification](https://yaml.org/type/bool.html) and should be interpreted as a `false` bool, not a `'off'` string. But indeed, their documentation don't make it clear. If you think it's safer, we could leave it as `off` and add a disable-line comment for the linter in this line. What do you think?

Reply:
I guess we can try with `false`, this is not a critical config